products/rhel8/profiles/stig_gui.profile

```diff
@@ -41,3 +41,8 @@ selections:
 
     # RHEL-08-040001
     - '!package_libreport-plugin-rhtsupport_removed'
+
+    # RHEL-08-040284
+    # Limiting user namespaces cause issues with user apps, such as Firefox and Cheese
+    # https://issues.redhat.com/browse/RHEL-10416
+    - sysctl_user_max_user_namespaces
```

Comment:
```suggestion
    - '!sysctl_user_max_user_namespaces'
```
Since you are unselecting should there be a `!` at the start?

Reply:
Ups!! : ) Thanks Matthew, it was an attention issue on my side. Fixing.

Reply:
Fixed.